src/codegen/src/main/scala/PySparkWrapperTest.scala

```diff
@@ -101,6 +101,10 @@ abstract class PySparkWrapperTest(entryPoint: PipelineStage,
           |        model = my$entryPointName.fit(data)
           |        self.assertNotEqual(model, None)""".stripMargin
 
+  protected def tryMultiColumnFitTemplate(entryPointName: String, model: String) =
```

Comment:
Could you explain why this code is needed? Shouldnt the code ion the case statement be sufficient? Thanks

Reply:
The model param does not exist for MultiColumnAdapter. One option was to change tryTemplate to take "model" or "baseStage" as input and handle it accordingly. I decided this fix would avoid short term errors more easily, and since I am pushing to rewrite this portion of the code base with Maureen and Eli, that would be preferred.

Reply:
Thanks for the snappy review!